client/vscode/tests/vsce.test.ts

```diff
@@ -139,19 +139,21 @@ describe('VS Code extension', () => {
             throw new Error('Timeout waiting for search results to render after viewing repo page')
         }
 
-        // Open remote file from search results
-        try {
-            await searchPanelFrame.waitForSelector('.test-search-result strong', { visible: true })
-            await searchPanelFrame.click('.test-search-result strong', { delay: 100 })
-        } catch {
-            throw new Error('Timeout waiting for search results to render after nevigating back from repo display page')
-        }
-
-        // Look for file title
-        const remoteFileTitle = await vsCodeDriver.page.title()
-        if (!remoteFileTitle.includes('bool_or_string_test.go')) {
-            throw new Error('Timeout waiting for remote file to render')
-        }
+        // // Open remote file from search results
```

Comment:
I think whenever we push a block of commented code to `main`, it's nice to state the reason in a comment. It saves a trip to git blame for anyone who's interested.
```suggestion
        // This will be fixed and re-enabled in https://github.com/sourcegraph/sourcegraph/issues/40366
        // // Open remote file from search results
```